src/logger_dispatch.jl

```diff
@@ -26,6 +26,18 @@ function preprocess(name, val::T, data) where T
     data
 end
 
+"""
+    tb_logged_propertynames(val::Any)
+
+Returns a tuple with the name of the fields of the structure `val` that 
+should be logged to TensorBoard. This function should be overridden when
+you want TensorBoard to ignore some fields in a structure when logging 
+it. The default behaviour is to return the  same result as `propertynames`. 
+
+See also: [`Base.propertynames`](@ref)
+"""
+tb_logged_propertynames(val::Type) = propertynames(val)
```

Comment:
I suggest just calling this `logable_propertynames`.
Since anyone overloading it will overload it as `TensorBoard.logable_propertynames(::Type{<:MyType}}) = ...`
So no need to give it the ugly name prefix.

Reply:
good point